server/parser/src/main/antlr4/org/eclipse/lsp/cobol/core/CobolParser.g4

```diff
@@ -2181,18 +2173,20 @@ length
    ;
 
 argument
-   : arithmeticExpression
-   | TRAILING | LEADING
+   : ALL
+   | arithmeticExpression
+   | TRAILING
+   | LEADING
```

Comment:
seems this is could be used only as function parameter. Couldn't find document explicitly for this. But did found   below code snippet
` MOVE FUNCTION TRIM('TEST123   ', TRAILING)
             TO TRIM-OUT-VAL
           MOVE FUNCTION TRIM('   TEST123', LEADING)
             TO TRIM-OUT-VAL`

Reply:
If this true than , will have to add a semantics check here